apps/permissions/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.permission_home, name="home_permission"),
+    path('create_permission/', views.permission_create,
+         name="create_permission"),
+    path('list_permissions/', views.permission_list,
+         name="list_permissions"),
+    path('update_permission/<str:pk>/', views.permission_update,
+         name="update_permission"),
+    path('delete_permission/<str:pk>/', views.permission_delete,
+         name="delete_permission"),
+    path('list_users/', views.permission_list_users,
+         name='list_users_permissions'),
+    path('list_zones/', views.permission_list_zones,
+         name='list_zones_permissions'),
+    path('update_user_permissions/<str:pk>/', views.permission_user_update,
+         name="update_user_permissions"),
+    path('update_zone_permissions/<str:pk>/', views.permission_zone_update,
```

Comment:
In urls it is a good practice to use hyphen '-' over underscore '_' especially as whole app utilizes '-'
ex.
"list_zones/" -> "list-zones/"
or example from this app elsewhere
```py
    path('companies/<int:pk>/update/', views.company_update, name='company_update'),
    path('companies/<int:pk>/delete/', views.company_delete, name='company_delete'),
    path('companies/assign_user_to_company/', views.assign_user_to_company, name='assign_user_to_company'),
    path('companies/by-id/<int:pk>/', views.company_by_id, name='company_by_id'),
    ```